poolapi/apijson.go

```diff
@@ -0,0 +1,41 @@
+package poolapi
+
+type Response struct {
+	Status  string      `json:"status"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data,omitempty"`
```

Comment:
I think the data is always a json object with json strings as values.  In go that would be a map[string]string.